coderd/files_test.go

```diff
@@ -17,26 +18,38 @@ func TestPostFiles(t *testing.T) {
 		t.Parallel()
 		client := coderdtest.New(t, nil)
 		_ = coderdtest.CreateFirstUser(t, client)
-		_, err := client.Upload(context.Background(), "bad", []byte{'a'})
+
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
```

Comment:
As much as possible, I tried to ensure contexts are initialized _after_ test environment has been set up (e.g. post- coderd client, user creation etc). This ensure the timeout is applied only to the part being tested. The `coderdtest` methods have their own timeouts.